tests/utils/fs/index.js

```diff
@@ -17,6 +18,12 @@ function getTmpFilePath(fileName) {
   return path.join(getTmpDirPath(), fileName);
 }
 
+function createFile(name, content) {
+  const filePath = getTmpFilePath(name);
+  fse.ensureFileSync(filePath, content);
```

Comment:
If I see correctly, the version of `fs-extra` we depend on, do not support `content` argument -> https://github.com/jprichardson/node-fs-extra/tree/0.26.7#ensurefilefile-callback

Reply:
Wow! Good catch!